rclpy/src/rclpy/node.hpp

```diff
@@ -42,6 +42,33 @@ get_node_name(py::capsule pynode);
  */
 const char *
 get_node_namespace(py::capsule pynode);
+
+/// Count publishers for a topic.
+/*
+ * Raises ValueError if pynode is not a node capsule
+ * Raises RCLError if rcl_count_publishers doesn't work
```

Comment:
@ahcorde :smile:, would you consider: 
```suggestion
 * Raises RCLError if an error occurs in rcl
```
? Same below.